website/docs/d/vapp_vm.html.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_vm"
+sidebar_current: "docs-vcd-datasource-vapp-vm"
+description: |-
+  Provides a vCloud Director VM data source. This can be used to access VMs within a vApp.
+---
+
+# vcd\_vapp\_vm
+
+Provides a vCloud Director VM data source. This can be used to access VMs within a vApp.
+
+Supported in provider *v2.6+*
+
+## Example Usage
+
+```hcl
+
+data "vcd_vapp" "web" {
+  name= "web"
+}
+
+data "vcd_vapp_vm" "web1" {
+  vapp_name     = "${vcd_vapp.web.name}"
+  name          = "web1"
+}
+
+output "vm" {
+  value = data.vcd_vapp_vm.web1
+}
+```
+
+Sample output:
+```
+vm = {
+  "computer_name" = "TestVM"
+  "cpu_cores" = 1
+  "cpus" = 2
+  "description" = "This OVA provides a minimal installed profile of PhotonOS. Default password for root user is changeme"
+  "disk" = []
+  "guest_properties" = {}
+  "href" = "https://my-vcd.org/api/vApp/vm-ecb449a2-0b11-494d-bbc7-6ae2f2ff9b82"
+  "id" = "urn:vcloud:vm:ecb449a2-0b11-494d-bbc7-6ae2f2ff9b82"
+  "memory" = 1024
+  "metadata" = {
+    "vm_metadata" = "VM Metadata."
+  }
+  "name" = "vm-datacloud"
+  "network" = [
+    {
+      "ip" = "192.168.2.10"
+      "ip_allocation_mode" = "MANUAL"
+      "is_primary" = true
+      "mac" = "00:50:56:29:08:89"
+      "name" = "net-datacloud-r"
+      "type" = "org"
+    },
+  ]
+  "org" = "datacloud"
+  "storage_profile" = "*"
+  "vapp_name" = "vapp-datacloud"
+  "vdc" = "vdc-datacloud"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
+* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
+* `vapp_name` - (Required) The vApp this VM belongs to.
+* `name` - (Required) A name for the VM, unique within the vApp 
+
+## Attribute reference
```

Comment:
Please confirm with an org admin user whether there are any fields which values are *not* returned and which we should mark with the newly introduced `(System Admin)` flag in the docs. It doesn't look like there should be in this data source, but just in case.

Reply:
Tests with Org Admin pass without errors